crates/common/builder/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "ream-builder"
+authors.workspace = true
+edition.workspace = true
+keywords.workspace = true
+license.workspace = true
+readme.workspace = true
+repository.workspace = true
+rust-version.workspace = true
+version.workspace = true
+
+[dependencies]
+alloy-primitives.workspace = true
+anyhow.workspace = true
+ethereum_hashing.workspace = true
+ethereum_serde_utils.workspace = true
+ethereum_ssz.workspace = true
+ethereum_ssz_derive.workspace = true
+reqwest.workspace = true
+serde.workspace = true
+serde_json.workspace = true
+ssz_types.workspace = true
+tree_hash.workspace = true
+tree_hash_derive.workspace = true
+url.workspace = true
+
+# ream dependencies
+ream-bls.workspace = true
+ream-consensus.workspace = true
```

Comment:
https://github.com/ReamLabs/ream/pull/584#pullrequestreview-2940463870

> I left some feedback, I don't think it makes sense for us to build a builder, so we should only be implementing the validator specs to support mev-relays no?

^ this earlier comment seems to have been missed.

I don't think there is really a reason to make this it's own crate, it is minimal enough it might as well be a module in the validator crate. It doesn't seem like there will be a usecase outside of the validator as well.